src/components/BookForm.js

```diff
@@ -29,29 +30,33 @@ function BookForm() {
 
   return (
     <div>
+      <hr style={{ borderTop: '1px solid #e8e8e8', marginTop: '40px' }} />
+
       <h2>Add New Book</h2>
-      <input
-        type="text"
-        placeholder="Title"
-        value={title}
-        onChange={(e) => setTitle(e.target.value)}
-      />
-      <input
-        type="text"
-        placeholder="Author"
-        value={author}
-        onChange={(e) => setAuthor(e.target.value)}
-      />
-      <select value={category} onChange={(e) => setCategory(e.target.value)}>
-        {categories.map((cat) => (
-          <option key={cat} value={cat}>
-            {cat}
-          </option>
-        ))}
-      </select>
-      <button type="button" onClick={handleAddBook}>
-        Add Book
-      </button>
+      <div className="input-container">
+        <input
+          type="text"
+          placeholder="Title"
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+        <input
+          type="text"
+          placeholder="Author"
+          value={author}
+          onChange={(e) => setAuthor(e.target.value)}
+        />
+        <select value={category} onChange={(e) => setCategory(e.target.value)}>
+          {categories.map((cat) => (
+            <option key={cat} value={cat}>
+              {cat}
+            </option>
+          ))}
+        </select>
+        <button className="add-book-button" type="button" onClick={handleAddBook}>
+          <span className="button-text">Add Book</span>
+        </button>
+      </div>
```

Comment:
Original design
![image](https://github.com/DevKaranJ/bookstore/assets/78251927/1556abe7-4088-4e11-a818-76634a03739b)

Your design
![image](https://github.com/DevKaranJ/bookstore/assets/78251927/50b0a43f-6fd8-4bab-89bc-69e0e4aaedd4)

Please adjust your design to match Zeplin's design (this includes all elements + the same styling)

- Spacing between elements (margin/padding)